packages/smart-forms-renderer/src/components/FormComponents/ChoiceItems/ChoiceRadioSingle.tsx

```diff
@@ -52,7 +52,7 @@ function ChoiceRadioSingle(props: ChoiceRadioSingleProps) {
       }}
       disabled={isHtmlDisabled}
       value={value}
-      control={<StandardRadio size="small" readOnly={isHtmlReadOnly} />}
+      control={<StandardRadio data-test={label} size="small" readOnly={isHtmlReadOnly} />}
```

Comment:
it's too dangerous to make data-test={label}, use data-test={`label-${label}`} instead